modules/ROOT/pages/import-api-specification-exchange.adoc

```diff
@@ -32,7 +32,7 @@ To list all available APIs, type a `*` character in the search bar and press Ent
 
 Studio scaffolds your API specification. If you choose not to create flows out of your API specification, unselect *Scaffold Flows From These API Specifications*.
 
-Manage the API specifications linked to Exchange in your project from the API specification project management view by clicking the *Manage Mule Project APIs* icon (image:manage-project-api-icon.png["A stylized letter i with a curved roof over it" 2%,2%]) in the task bar.
+Manage the API specifications linked to Exchange in your project from the API specification project management view by clicking the *Manage Mule Project APIs* icon (image:manage-project-api-icon.png["A stylized letter with a curved roof over it" 2%,2%]) in the task bar.
```

Comment:
Wasn't the original text correct? The icon is the letter **i** with a roof on top.

Reply:
You're right! Thanks!